src/stories/Starting.stories.mdx

```diff
@@ -0,0 +1,196 @@
+import { Meta } from '@storybook/addon-docs';
+
+import github from './assets/github.svg';
+import npm from './assets/npm.svg';
+import figma from './assets/figma.svg';
+
+
+
+<Meta title="Starting" />
+
+<style>
```

Comment:
Externalize style for futur

Reply:
I didn't quite understand, why externalize style of storybook ?